en/sql/function/json_fn.rst

```diff
@@ -369,3 +369,111 @@ Accepting any paths that contain json array indexes will filter out '$.b'
       json_search('{"a":["a","b"],"b":"a","c":["a"], "d":{"e":["a"]}}', 'all', 'a', null, '$**[*]')
     ======================
       "["$.a[0]","$.c[0]","$.d.e[0]"]"
+
+JSON_EXTRACT
+===================================
+
+.. function:: JSON_EXTRACT (json_doc, json path [ , json path] ...)
+
+  Returns json elements from the json_doc, that are addressed by the given paths.
+  If json path argments contain wildcards, all elements that are addressed by a path compatible with the wildcards-containg json path are gathered in a resulting json array. 
+  A single json element is returned if no wildcards are used in the given json paths and a single element is found, otherwise the json elements found are wrapped in a json array.
+  Raises an error if a json path is NULL.  
+  Returns NULL if no elements are found or if json_doc arguments is invalid.
```

Comment:
Returns null of json_doc argument is invalid? Shouldn't be an error?

Please note you have a typo `if json_doc arguments is `